index.js

```diff
@@ -1,34 +1,46 @@
-var ispnum = function(num) {
```

Comment:
Function expression should be replaced with function declaration. It allows to use function before definition, also it  decrease problems related to code ordering.

Also function naming should be more understandable (for example isPrimeNumber).